### PR TITLE
Adding delete group button

### PR DESCRIPTION
**Pull request by vifrn**

### CODE REVIEW

File: src/main/java/com/tibagni/logviewer/LogViewerPresenter.java [hunk 1]
```diff
@@ -133,6 +133,15 @@ public void removeFilters(String group, int[] indices) {
     }
   }
 
+  @Override
+  public void removeGroup(String group){
+    if(!StringUtils.isEmpty(group)){
+      filters.remove(group);
```
Comment (tibagni) [inline src/main/java/com/tibagni/logviewer/LogViewerPresenter.java]: Before actually removing the filter, you can check if it has been changed by the user ('haveFiltersChanged') and, if so, as if the user wants to save it before removing.
Check example on 'finishing'

File: src/main/java/com/tibagni/logviewer/filter/FiltersList.java [hunk 2]
```diff
@@ -145,6 +147,7 @@ public FilterUIGroup(String groupName, Filter[] filters) {
       addBtn = new FlatButton("+");
       saveBtn = new FlatButton("Save");
       saveBtn.setVisible(false);
+      deleteBtn = new FlatButton("x");
```
Comment (tibagni) [inline src/main/java/com/tibagni/logviewer/filter/FiltersList.java]: https://en.wikipedia.org/wiki/X_mark

File: src/main/java/com/tibagni/logviewer/LogViewerPresenter.java (hunk 1 above)
Comment (vifrn) [reply src/main/java/com/tibagni/logviewer/LogViewerPresenter.java]: Done

File: src/main/java/com/tibagni/logviewer/filter/FiltersList.java (hunk 2 above)
Comment (vifrn) [reply src/main/java/com/tibagni/logviewer/filter/FiltersList.java]: There are many 'x' characters in this link. I chose the one I think looks better.

**Pull request merged**